src/configs/nim.rs

```diff
@@ -0,0 +1,23 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct NimConfig<'a> {
+    pub symbol: SegmentConfig<'a>,
+    pub version: SegmentConfig<'a>,
+    pub style: Style,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for NimConfig<'a> {
+    fn new() -> Self {
+        NimConfig {
+            symbol: SegmentConfig::new("👑 v"),
```

Comment:
The prefix shouldn't be part of a symbol:
```suggestion
            symbol: SegmentConfig::new("👑 "),
```

Reply:
How can I keep it configurable then? Should I add prefix/suffix support?

Reply:
It shouldn't be configurable. Version should always start with `v` prefix because that is starship's convention. Prefix and suffix mean different thing: https://starship.rs/config/?#terminology

Reply:
In general we always prefix the version with a `v` and this is not configurable. Also having the `v` in the symbol will cause issues when we move to the new format string configuration. This also doesn't match what is documented in `docs/config/README.md` so might cause some confusion for the end users.